es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/clients/RestHighLevelClientAdapter.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ *     Beshu Limited all rights reserved
+ */
+package tech.beshu.ror.proxy.es.clients
+
+import java.util
+
+import monix.eval.Task
+import org.elasticsearch.action.admin.cluster.health.{ClusterHealthRequest, ClusterHealthResponse}
+import org.elasticsearch.action.admin.indices.alias.get.GetAliasesRequest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)